ush/prdgen/subset_ifi_304m.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I would suggest renaming this file to rrfs_subset_ifi_304m.sh, since the other scripts in ush/prdgen start with "rrfs".